packages/thumbprint-atomic/package.json

```diff
@@ -16,7 +16,7 @@
     "scripts": {
         "start": "yarn clear && yarn generateCss",
         "clear": "rm -rf dist",
-        "generateCss": "mkdir -p dist && node-sass -q src/atomic.scss dist/atomic.css --importer=./node-sass-importer.js && postcss dist/atomic.css --map --use autoprefixer css-mqpacker cssnano --replace",
+        "generateCss": "mkdir -p dist && ./generate-css.js src/atomic.scss dist/atomic.css && postcss dist/atomic.css --map --use autoprefixer css-mqpacker cssnano --replace",
```

Comment:
Looks like the `src/atomic.scss dist/atomic.css` part can be removed since `./generate-css.js` doesn't accept args?

Reply:
Done